site/docs/components/border-layout/examples.mdx

```diff
@@ -25,17 +25,17 @@ By default, there are no gaps between regions.
 
 You can manually define the size of regions. This example shows how to set the West and East regions to a fixed width, with a responsive Center region.
 
-This layout would work well where a fixed-width side navigation is required, alongside responsive page content.
+This layout would work well where a fixed-width side navigation is necessary, alongside responsive page content.
 
 </LivePreview>
 
 <LivePreview componentName="border-layout" exampleName="GapBetweenRegions" >
 
 ### Gap between regions
 
-A gap can be added between each region. The `gap` property is a multiplier that aligns to the [Salt spacing system](/salt/foundations/spacing).
+You can add a gap between each region. The `gap` prop is a multiplier that aligns to the [Salt spacing system](/salt/foundations/spacing).
 
-- The standard gap between column and row elements in Salt is 3 times the base unit.
+- The standard gap between column and row elements in Salt is three times the base unit.
```

Comment:
The current content guide suggests this should stay as 3 since it's used mathematically. Has this changed?